contribute/documentation/README.md

```diff
@@ -1,7 +1,14 @@
 # Contribute to our documentation
 
-Welcome. We're glad you're here to help make our technical documentation even better.
+Welcome! We're glad you're here.
 
-For our style guide and writing guidelines, see [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/).
+[Writers' Toolkit](https://grafana.com/docs/writers-toolkit/) is an open source project intended to empower both internal and external contributors to help make our technical documentation even better.  
 
-If you're interested in contributing to the Writers' Toolkit, refer to the [Writers' Toolkit](https://github.com/grafana/writers-toolkit) repository.
+Go to [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/) for the following resources:
+
+- **Contribution guidelines** - Learn how can you contribute to Grafana Labs documentation.
+- **Structure guidelines** - Learn how to organize and structure our documentation.
+- **Writing and style guide** - Guidelines that cover aspects of content such as style, tooling, workflows, markdown syntax, and more.
+- **Build and review guide** - Instructions for testing, linting, validating, and backporting our documentation.
+
+If you're interested in contributing to our documentation, go to [Writers' Toolkit](https://github.com/grafana/writers-toolkit) repository for writers' resources.
```

Comment:
Following on from https://github.com/grafana/grafana/pull/90405#discussion_r1677414350.

I'm a bit confused by the "writers's resources" bit, but I'm no expert on grammar. Do the resources belong to the writers? I guess so?

Is it not "the Writers' Toolkit repository" rather than "Writers' Toolkit repository". I guess it could also be "Writers' Toolkit's repository" but that seems really off.

Is this easier to understand?
```suggestion
If you're interested in contributing to our writing resources, go to the [Writers' Toolkit](https://github.com/grafana/writers-toolkit) repository.
```

Reply:
Doc team discussion has preferred to omit "the" for some reason. See [Slack convo](https://raintank-corp.slack.com/archives/C01EK60DKC5/p1721848235399109).

Reply:
My thought was that it was to use "the Writer's Toolkit repository" but not "Writers' Toolkit". In the same way, I'd think it's fine to do "the Grafana Agent repository" but not "the Grafana Agent" or "Grafana Agent's repository" (no product possessives).

Reply:
> Following on from [#90405 (comment)](https://github.com/grafana/grafana/pull/90405#discussion_r1677414350).
> 
> I'm a bit confused by the "writers's resources" bit, but I'm no expert on grammar. Do the resources belong to the writers? I guess so?
> 
> Is it not "the Writers' Toolkit repository" rather than "Writers' Toolkit repository". I guess it could also be "Writers' Toolkit's repository" but that seems really off.
> 
> Is this easier to understand?

I rewrote this as:

If you're interested in contributing to our documentation, go to the [grafana/writers-toolkit repository](https://github.com/grafana/writers-toolkit) for relevant resources.

Reply:
That's much easier!